vuu-ui/showcase/src/App.tsx

```diff
@@ -100,34 +100,39 @@ export const App = ({ stories }: AppProps) => {
             revealSelected
             source={source}
           />
-          <Flexbox
-            className="ShowcaseContentContainer"
-            resizeable
-            style={{ flexDirection: "column", flex: "1 1 auto" }}
+          <ThemeProvider
```

Comment:
Having a ThemeProvider nested between a Flexbox and its child is breaking the layout - try resizing the panel on the left.
I need to make a change to the Layout components to allow ThemeProvider as a Layout warpper